services/daily_digest.py

```
@@ -0,0 +1,285 @@
+        .replace("*", "\\*")
+        .replace("_", "\\_")
+        .replace("`", "\\`")
+        .replace("[", "\\[")
+    )
+
+
+def _try_parse_json(s: str) -> dict[str, Any]:
+    try:
+        v = json.loads(s)
+        return v if isinstance(v, dict) else {"raw": v}
+    except Exception:
```

Ruff (0.15.17)

[warning] Do not catch blind exception: `Exception`

(BLE001)